Recipes/Recipes/ProfileModule/ProfileViewController.swift

```diff
@@ -3,12 +3,200 @@
 
 import UIKit
 
+private enum Constants {
```

Comment:
```suggestion
private enum Constants {
```
если это для ProfileViewController тогда внутри контролера перенеси как у других